requirements.txt

```diff
@@ -40,3 +40,8 @@ transformers>=4.40.0 ; python_version == '3.10'
 transformers>=4.41.2 ; python_version >= '3.11'
 trl>=0.9.4
 wandb>=0.16.4
+ruamel.yaml>=0.17.0
+# the below library should NOT be imported into any python files
+# it is for CLI usage ONLY
+yamllint>=1.35.1
+referencing>=0.35.0
```

Comment:
The rebase incorrectly added these lines back. They should be removed.

Reply:
sorry about that!

Reply:
Did you push a change to remove these 4 lines? The PR still contains them.

Reply:
I just did.